velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -33,11 +44,13 @@ class NthValueTest : public WindowTestBase {
         makeFlatVector<int64_t>(size, [](auto row) { return row % 3 + 1; }),
         typeValues(type, size),
     });
-    testTwoColumnOverClauses({vectors}, "nth_value(c3, c2)");
-    testTwoColumnOverClauses({vectors}, "nth_value(c3, 1)");
-    testTwoColumnOverClauses({vectors}, "nth_value(c3, 5)");
+    testTwoColumnOverClauses({vectors}, "nth_value(c3, c2)", frameClause_);
```

Comment:
Since testTwoColumnOverClauses anyways tests over all clauses you can iterate over frames there itself, instead of iterating in this class.